src/main.js

```diff
@@ -1,32 +1,43 @@
 // Global Variables
-var currentGame = "";
-var gameModesSection = document.querySelector('#gameModes');
-var playerSide = document.querySelector('#playerSide');
-var characterOptions = document.querySelector('#characterOptions');
 var backButton = document.querySelector('#backButton');
+var characterOptions = document.querySelector('#characterOptions');
+var computerScore = document.querySelector('#computerScore');
+var currentGame = new Game;
```

Comment:
Do you need to set a new game if you do it in a later method? Maybe just instantiate an undefined variable.
`var currentGame;`